Tests/SwiftParserTest/LexerTests.swift

```diff
@@ -1080,6 +1080,41 @@ public class LexerTests: ParserTestCase {
     }
   }
 
+  func testUTF16Surrogates_1() {
+    // 0xD800 <= (UTF16 Surrogates value) <= 0xDFFF
+    let sourceBytes: [UInt8] = [0xED, 0xA0, 0x80]  // 0xD800
```

Comment:
Also a little nitpicky but I think we could improve clarity here by changing `// 0xD800` to `The bytes represent the code point 0xD800` (I think that code point is the correct term here, but I might be wrong).

Same below

Reply:
`Code point` is correct here, and I found that we normally suffix it with `U+`, for example, `U+D800`, when referring to code points.